ci/test_integration.sh

```diff
@@ -209,6 +209,17 @@ echo "==== feats_type=raw, token_types=bpe, model_conf.extract_feats_in_collect_
 ./run.sh --ngpu 0 --stage 10 --stop-stage 13 --skip-upload false --feats-type "raw" --token-type "bpe" \
     --feats_normalize "utterance_mvn" --lm-args "--max_epoch=1" --python "${python}" \
     --asr-args "--model_conf extract_feats_in_collect_stats=false --max_epoch=1"
+
+echo "==== use_k2, num_paths > nll_batch_size, feats_type=raw, token_types=bpe, model_conf.extract_feats_in_collect_stats=False, normalize=utt_mvn ==="
```

Comment:
FYI: now you should move these tests to ci/test_integration_espnet2.sh because of #3599.

Reply:
fixed by moving it to ci/test_integration_espnet2.sh.